examples/demo/Advanced/Date_range_editor_demo.py

```diff
@@ -2,34 +2,50 @@
 #  License: BSD Style.
 
 """
-A Traits UI editor that wraps a Qt calendar panel.
+**WARNING**
+
+  This demo might not work as expected and some documented features might be
+  missing.
+
```

Comment:
I put the warning here due to broken UI using qt. Because I'm on a Mac, the issue was really noticeable, but in the wider community this issue might be niche enough to not need a warning.

As for editor not being implemented in wx, I will add that to the docstring and fix a typo that I found.